### PR TITLE
Fix internal link error of #1506

### PR DESCRIPTION
**Pull request by tfiers**
Implementation of https://github.com/progit/progit2/pull/1506#issuecomment-683137148

### CODE REVIEW

**Comment by tfiers** [comment]
Alright, offending line in build from https://github.com/progit/progit2/pull/1506#issuecomment-683133393 is gone now.

File: book/03-git-branching/sections/nutshell.asc [hunk 1]
```diff
@@ -3,13 +3,13 @@
 
 To really understand the way Git does branching, we need to take a step back and examine how Git stores its data.
 
-As you may remember from <<ch01-getting-started#_what_is_git>>, Git doesn't store data as a series of changesets or differences, but instead as a series of _snapshots_.
+As you may remember from <<ch01-getting-started#section__what_is_git>>, Git doesn't store data as a series of changesets or differences, but instead as a series of _snapshots_.
```
Comment (HonkingGoose) [inline book/03-git-branching/sections/nutshell.asc:6]: Did you mean to add 2 underscores here: `#section__`?

Comment (HonkingGoose) [reply book/03-git-branching/sections/nutshell.asc:6]: The only other anchor tag in the book uses just 1 underscore after `#section_` so to be consistent I would only use 1 underscore. :smile:

Comment (HonkingGoose) [reply book/03-git-branching/sections/nutshell.asc:6]: See: https://raw.githubusercontent.com/progit/progit2/master/book/04-git-server/sections/gitlab.asc

**Comment by jnavila** [comment]
local test:
image

File: book/03-git-branching/sections/nutshell.asc (hunk 1 above)
Comment (tfiers) [reply book/03-git-branching/sections/nutshell.asc:6]: Yes I meant to use 2 underscores to distinguish the 'type' from the 'identifier'.

But good catch with the other ID; I'll change it to be consistent (but then without a leading underscore); i.e. `what_is_git_section`

Comment (HonkingGoose) [reply book/03-git-branching/sections/nutshell.asc:6]: @jnavila what do you think?

Comment (jnavila) [reply book/03-git-branching/sections/nutshell.asc:6]: I'm not purist, but just being consistent is great.

**Comment by ben** [comment]
Everybody happy here? I'm ready to merge.

**Review by HonkingGoose — APPROVED** [review]

**Comment by tfiers** [comment]
Thanks all for the help

**Pull request merged**